### ISSUE TITLE
Maass source misattribution

### ISSUE THREAD

**Issue by davidfarmer** [LMFDB/lmfdb#3798]
On the Maass forms source page:

https://beta.lmfdb.org/ModularForm/GL2/Q/Maass/Source

it omits to say that the level 1 data is from Holger Then.

**Comment by AndrewVSutherland** [comment]
@davidfarmer Fixing this is just a matter of editing the knowl

https://beta.lmfdb.org/knowledge/edit/rcs.source.maass

**Comment by davidfarmer** [comment]
I will update the attribution knowl once I am done checking the data, because I know there
will be other changes.

I also will use this issue as a catch-all to keep track of other issues I find and plan to fix
(I'll submit separate issues for things I will not fix as part of this immediate task.)

1.  The number of available coefficients as indicated in the 'Numc' field need not equal
the actual number of available coefficients.  Delete/ignore the 'Numc' field and use the
actual number of coefficients.

2.  Set to exactly 0 numbers which we know are 0 but which have very small values
in the database.  For example, the 18th coefficients in each of these:

https://beta.lmfdb.org/ModularForm/GL2/Q/Maass/4f622d0488aece3d9f000003
https://beta.lmfdb.org/ModularForm/GL2/Q/Maass/4f62419388aece3d9f000016

3. Insert glaring omissions, such as the missing blue dot at 6.64, 9 here:
https://beta.lmfdb.org/ModularForm/GL2/Q/Maass/BrowseGraph/1/10/0/10/

**Comment by davidfarmer** [comment]
There are 464 examples with nontrivial character.  Those might not be checked in this initial phase of the testing.

The Atkin-Lehner eigenvalues may not be checked in the initial phase, but possibly could be checked when doing the nontrivial characters.

**Comment by davidfarmer** [comment]
Search for level 5.  Notice some anomalies in the last 5 columns.  Most of those are due to to database problems, not due to a lack of available information.  Should be possible to fix those.

**Comment by davidfarmer** [comment]
More examples of tiny numbers which should be set to 0:

https://beta.lmfdb.org/ModularForm/GL2/Q/Maass/4f746bc488aece0ea700002b

**Comment by davidfarmer** [comment]
All of this has been done, except the Source knowl, for which I had a server error when editing.  Will submit a separate issue.

**Comment by edgarcosta** [comment]
You can now edit the source knowl.

**Comment by davidfarmer** [comment]
I have sent @AndrewVSutherland the information he needs to
make adjustments to the database. Once that is done, then I need to
modify the Reliability knowl (to omit reference to the Precision,
assuming Drew omits that field), and also delete the no longer used
Precision knowl.  

This should stay open until the data is revised and those knowls are updated.

**Comment by AndrewVSutherland** [comment]
Thanks for the data @davidfarmer!  It will take some time to integrate this into the LMFDB and make it available to users -- my plan is to create a completely new table with a new schema and to update or even completely replace the code that implements the current UI (the current code is very hard to maintain so my inclination is to replace it, which I think will be less work in the long run).

This will surely take more than a few "LMFDB Fridays" to finish, especially with ANTS coming up soon, but I would hope to complete this work before the end of July.

**Comment by davidfarmer** [comment]
All this seems to be done in a branch by Drew.
